stream-transformation-test/stream-transformation-it/src/test/java/uk/gov/justice/framework/tools/replay/StreamTransformationIT.java

```diff
@@ -179,11 +178,13 @@ private static DataSource initDatabase(final String dbUrlPropertyName,
         return dataSource;
     }
 
-    public boolean eventStoreTransformedEventPresent() throws SQLException {
+    private boolean eventStoreTransformedEventPresent() throws SQLException {
         final Stream<Event> eventLogs = EVENT_LOG_JDBC_REPOSITORY.findAll();
         final Optional<Event> event = eventLogs.filter(item -> item.getStreamId().equals(STREAM_ID)).findFirst();
-
-        return event.isPresent();
+        if (event.isPresent() && event.get().getName().equals("sample.events.transformedName")) {
```

Comment:
you can just return the line, don't need to return true on the IF and false on the ELSE.